explorer/client/src/components/transaction-card/RecentTxCard.tsx

```diff
@@ -134,6 +139,11 @@ function LatestTxView({
                                     alttext={truncate(tx.txId, TRUNCATE_LENGTH)}
                                 />
                             </div>
+                            {tx.timestamp_ms && (
```

Comment:
can we unify this with the copy of it in `TXForId` ?  i think realistically we will want to repeat timestamp display in quite a few places and we want to keep them consistent.